docarray/documents/text.py

```diff
@@ -79,3 +95,19 @@ def validate(
         if isinstance(value, str):
             value = cls(text=value)
         return super().validate(value)
+
+    def __eq__(self, other: Any) -> bool:
+        if isinstance(other, str):
+            return self.text == other
+        else:
+            # BaseModel has a default equality
+            return super().__eq__(other)
+
+    def __contains__(self, item: str):
```

Comment:
can u add to the docstring how this look lile ?

Reply:
what do u mean?